examplebroker/broker.go

```diff
@@ -626,9 +626,13 @@ func (b *Broker) IsAuthenticated(ctx context.Context, sessionID, authenticationD
 
 	access, data = b.handleIsAuthenticated(ctx, sessionInfo, authData)
 	if access == auth.Granted && sessionInfo.currentAuthStep < sessionInfo.neededAuthSteps {
+		data = ""
+		if sessionInfo.pwdChange != noReset && sessionInfo.sessionMode == auth.SessionModeLogin {
+			data = fmt.Sprintf(`{"message": "Password reset, %d step(s) missing"}`,
+				sessionInfo.neededAuthSteps-sessionInfo.currentAuthStep)
+		}
```

Comment:
Is showing this message the reason for this change? I can understand why we would like to inform the user that something went wrong already instead of waiting for the end of the process, but is it worth it?

Also, if we decide that it is, then we should improve this message. Saying that `n steps are missing` doesn't inform much to the end user, I think.

Reply:
This is just in the example broker, but that's for testing purposes only...

Also I wanted to have it in a specific case only, and this looked a good place to be (not to have to change all the tests).